### DISCUSSION TITLE
React Native - New Renderer Interop Layer

### DISCUSSION THREAD

**Discussion by alexhernandez** [reactwg/react-native-releases#86] [Announcements]
One Challenge w/ the RN Interop layer described [here](https://github.com/reactwg/react-native-new-architecture/discussions/135) is figuring out what name should be used inside `unstable_reactLegacyComponentNames` since the package name listed in `package.json` does not work.

For visual elements like React Native Picker its easier to identity since the UI gives you the key inside a pink box, but what about other libs that don't have any visual elements?

Is there an easier way to figure out what should be placed inside my `react-native.config.js` for both iOS and Android?

Example: 
For React Native Cookies or React Native Netinfo, where should a developer go to identify the key to add inside `unstable_reactLegacyComponentNames` for both iOS and Android?

**Comment by alexhernandez** [comment]
@kelset @cipolleschi curious what your thoughts are

**Reply by cipolleschi** [reply]
Can you keep this question in the same discussion of the interop layer --> https://github.com/reactwg/react-native-new-architecture/discussions/135?

It would be easier to have all of them in the same place.

**Reply by alexhernandez** [reply]
Unfortunately I tried, but seems like it's closed to collaborators only. Could I be added?

**Reply by cipolleschi** [reply]
yes, there is a form hereyes, there is a form [here](https://github.com/reactwg/react-native-new-architecture#how-to-join-the-working-group)

**Reply by alexhernandez** [reply]
Thanks, form submitted!
